test/tools/router_check/router.cc

```diff
@@ -84,6 +85,18 @@ RouterCheckTool RouterCheckTool::create(const std::string& router_config_file,
                          std::move(api), Coverage(route_config));
 }
 
+// Set a unique number as the name for each route for detecting missing tests during the coverage
+// check.
+void RouterCheckTool::assignUniqueRouteNames(envoy::api::v2::RouteConfiguration& route_config) {
+  uint64_t route_name = 0;
```

Comment:
see if we can just use unique guid strings instead of numbers to avoid number to string conversion.

Reply:
/wait

Reply:
Switched to using the existing UUID generator from Runtime. I made had to switch to a less efficient way of detecting the missing tests in printMissingTests, as the two sets of UUIDs are sorted in a different order.